src/test/config/kind/common-values.yaml

```diff
@@ -53,3 +53,11 @@ monitoring:
     createDashboards: true
     dashboardLabels:
       grafana_dashboard: dc_monitoring
+
+synchrony:
+  enabled: true
+  resources:
+    container:
+      requests:
+        cpu: 20m
```

Comment:
This will alow schedule synchrony pod, and ActiveProcessorCount will be set to 1.